bentoml/yatai/deployment/azure_functions/Dockerfile

```diff
@@ -6,5 +6,8 @@ FROM bentoml/azure-functions:$BENTOML_VERSION-py$PYTHON_VERSION
 # Copying BentoService to Azure function specific directory
 COPY . /home/site/wwwroot
 
-# Install additional pip dependencies inside bundled_pip_dependencies dir
+# Install pip dependencies
 RUN if [ -f /home/site/wwwroot/bentoml-init.sh ]; then /bin/bash -c /home/site/wwwroot/bentoml-init.sh; fi
+
+# Install bundled bundled_pip_dependencies
+RUN if [ -d /home/site/wwwroot/bundled_pip_dependencies ]; then pip install -U /home/site/wwwroot/bundled_pip_dependencies/* ;fi
```

Comment:
I'm more inclined to move this back to `bentoml-init.sh`, although need to double-check if that will break our docker build cache and slow down repeat docker builds

Reply:
I agree on moving this back to the `bentoml-init.sh` file.  

Let me explore about the docker build cache. Also, if users have different versions of packages in the directory, docker will have to run that layer anyway